js/libs/swiftclick.js

```diff
@@ -97,32 +97,20 @@ function SwiftClick (contextEl)
 	function touchEndHandler (event)
 	{
 		var targetEl = event.target,
-			touchend,
-			allowFurtherEventsWhenCancellingSyntheticClick = true;
+			touchend;
 
 		targetEl.removeEventListener("touchend", touchEndHandler, false);
 
 		touchend = event.changedTouches[0];
 
-		// cancel the touch if the node type is unacceptable (not in the dictionary), or if the touchpoint position has drifted significantly.
-		if (//!_shouldSynthesizeClickEvent ||
+		// bail out if the touchpoint position has drifted significantly, user is not trying to click.
+		if (
 			Math.abs(touchend.pageX - _touchStartPoint.x) > _self.options.maxTouchDrift ||
 			Math.abs(touchend.pageY - _touchStartPoint.y) > _self.options.maxTouchDrift ||
 			Math.abs(getScrollPoint().x - _scrollStartPoint.x) > _self.options.maxTouchDrift ||
 			Math.abs(getScrollPoint().y - _scrollStartPoint.y) > _self.options.maxTouchDrift)
 		{
-			// stop further events if we are already tracking.
-			if (_currentlyTrackingTouch)
-			{
-				event.stopPropagation();
-				event.preventDefault();
-				allowFurtherEventsWhenCancellingSyntheticClick = false;
-			}
-
-			// reset vars to default state before returning early, effectively cancelling the creation of a synthetic click event.
-			_currentlyTrackingTouch = false;
-			//_shouldSynthesizeClickEvent = true;
-			return allowFurtherEventsWhenCancellingSyntheticClick;
+			return true;
```

Comment:
@prashantpalikhe because here you removed `_currentlyTrackingTouch = false;`, after this click users will no longer be able to click, as the `touchStartHandler` will simply `return true;`.

Please remove `_currentlyTrackingTouch = false;` from line 126 and place it somewhere near the start of the `touchEndHandler`, such as line 105.